src/Umbraco.Deploy.Contrib.Connectors/ValueConnectors/NestedContentValueConnector.cs

```diff
@@ -8,22 +8,32 @@
 using Umbraco.Core.Logging;
 using Umbraco.Core.Models;
 using Umbraco.Core.Services;
+using Umbraco.Deploy.Connectors;
+using Umbraco.Deploy.Connectors.ValueConnectors;
 using Umbraco.Deploy.Connectors.ValueConnectors.Services;
+using Umbraco.Deploy.Core;
 
 namespace Umbraco.Deploy.Contrib.Connectors.ValueConnectors
 {
     /// <summary>
     /// A Deploy connector for the NestedContent property editor
     /// </summary>
-    public class NestedContentValueConnector : IValueConnector
+    public class NestedContentValueConnector : ValueConnectorBase
     {
         private readonly IContentTypeService _contentTypeService;
         private readonly Lazy<ValueConnectorCollection> _valueConnectorsLazy;
         private readonly ILogger _logger;
 
-        // Our.Umbraco.NestedContent is the original NestedContent package
-        // Umbraco.NestedContent is Core NestedContent (introduced in v7.7)
-        public virtual IEnumerable<string> PropertyEditorAliases => new[] { "Our.Umbraco.NestedContent", "Umbraco.NestedContent" };
+        /// <inheritdoc />
+        /// <remarks>
+        /// Our.Umbraco.NestedContent is the original NestedContent package
+        ///  Umbraco.NestedContent is Core NestedContent (introduced in v7.7)
+        /// </remarks>
+        public override IEnumerable<string> PropertyEditorAliases { get; } = new[]
+        {
+            "Our.Umbraco.NestedContent",
```

Comment:
This editor is also only supported in Umbraco 7, so it should be safe to remove:

```suggestion
```